.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,11 +4,10 @@
 
 ## Changes
 
-<!-- List the changes done to fix a bug or introduce a new feature.Please note both user-facing changes and changes to internal API's here -->
+<!-- List the technical changes done to fix a bug or introduce a new feature. -->
 
 ## Checklist
 
-<!-- Note: Only user-facing changes require a changelog entry. Internal-only API changes do not require a changelog entry. Changes in documentation do not require a changelog entry. -->
-
--   [ ] All tests pass locally
--   [ ] I have created / updated documentation for this (if applicable)
+-   [ ] I have checked `make build` works locally.
+-   [ ] The `make precommit-check` passes without failures.
```

Comment:
I feel like this is irrelevant due *CI* current flow and nature where inevitably we'll witness changes if not run before ... these steps should be as easy as possible and stuff that happens anyway within the MR maybe create unnecessary friction? Heck I am thinking about removing both and just explicitly state if CI is not happy, neither are we (suggesting both commands, but not making too much fuss about these as CI runs the whole thing anyway) ... still, not a strong no, just a wonder, I don't see myself running neither commands for our own PRs because I know CI runs those anyway.